src/misc/support.cpp

```diff
@@ -313,22 +313,19 @@ void E_Exit(const char *format, ...)
 	throw(e_exit_buf);
 }
 
+/* Overloaded function to handle different return types of POSIX and GNU strerror_r variants */
+const char *strerror_result(int retval, const char *buf) {return retval == 0 ? buf : NULL;}
+const char *strerror_result(const char *retval, const char *buf) {return retval;}
+
```

Comment:
Thanks for this PR!
Can you adjust `NULL` --> `nullptr`
and then in your source tree, run `./scripts/format-commit.sh`.

Reply:
Done. :)

Reply:
Inbound